corehq/apps/style/static/app_manager/less/new_appmanager/table.less

```diff
@@ -61,3 +61,8 @@
     line-height: 30px;
   }
 }
+
+.full-width-select2 .select2-container {
```

Comment:
Maybe when v2 is released we can get rid of the various select2 classes in forms.less, that'd be nice...